plugin/src/test/kotlin/com/exactpro/th2/gradle/BaseTh2PluginTest.kt

```diff
@@ -85,6 +95,50 @@ internal class BaseTh2PluginTest {
         }
     }
 
+    @Test
+    fun `applies license configuration from build script`(
+        @TempDir tempDir: Path,
+    ) {
+        val project =
+            ProjectBuilder.builder().build()
+
+        val customAllowedLicense = "file:test-allowed-licenses.json"
+        val customLicenseNormalizer =
+            tempDir.resolve("test-license-normalizer-bundle.json").apply {
+                writeText("{}")
+            }.absolutePathString()
+        project.extensions.extraProperties.set(TH2_LICENCE_ALLOW_LICENCE_URL_PROP, customAllowedLicense)
+        project.extensions.extraProperties.set(TH2_LICENCE_LICENSE_NORMALIZER_BUNDLE_PATH_PROP, customLicenseNormalizer)
+        project.pluginManager.apply("com.exactpro.th2.gradle.base")
+
+        project.run {
+            assertAll(
```

Comment:
note: looks like we have only one assertion here. Don't think the outer "assertAll" is needed here